src/background_remover.cc

```diff
@@ -268,8 +268,38 @@ static cv::Mat resizeAndPadTo(const cv::Mat &frame, int targetw, int targeth, Pa
     return ret;
 }
 
-void BackgroundRemover::maskBackground(cv::Mat &frame /* rgb */, const cv::Mat &maskImage /* rgb */,
-                                       bool do_blur_mask, Timing &t) {
+void BackgroundRemover::blendPixels(const cv::Vec3b &a,
```

Comment:
This (and the following) function don’t seem to require access to `this`, so they could be declared static (not: a static member function). In fact, this being a method prevents the compiler from inlining this.